.env.example

```diff
@@ -0,0 +1,5 @@
+REACT_APP_SERVER_URL=http://localhost:5001
+REACT_APP_BASE_URL=http://localhost:5001
+REACT_APP_AUTH_URL=http://localhost:5001/auth
+REACT_APP_NOTIFICATION_URL=http://localhost:5001/notification
+REACT_APP_FILE_SERVICE_URL=http://localhost:5001/files
```

Comment:
new extra line needed at the end

Reply:
is there linter error?

Reply:
there is no @sasuntitan jan, need we add some lint configs included the one for this case or we are good with ignoring extra lines?